signing/secp256k1/impl/src/main/java/tech/pegasys/signers/secp256k1/multikey/MultiKeySignerProvider.java

```diff
@@ -144,6 +147,17 @@ public Signer createSigner(final FileBasedSigningMetadataFile metadataFile) {
     }
   }
 
+  @Override
+  public Signer createSigner(final RawSigningMetadataFile metadataFile) {
+    try {
+      final Credentials credentials = Credentials.create(metadataFile.getPrivKey());
+      return new CredentialSigner(credentials);
+    } catch (final Exception e) {
+      LOG.error("Unable to construct Raw signer from " + metadataFile.getFilename());
```

Comment:
nit: why use capital R for Raw? seems odd

Reply:
changed to lower case